app/assets/javascripts/course/experience_points_disbursement.js

```diff
@@ -0,0 +1,16 @@
+(function($) {
+  'use strict';
+  $(document).on('page:load ready', function() {
+    var $disbursementPage = $('.course-experience-points-disbursement.new');
+    if (!$disbursementPage.length) { return; }
+    $('#experience-points-disbursement-copy-button').on('click', function(event) {
```

Comment:
@kxmbrian Can you factor out the handler and define it as a separate method ? The structure will be like:
```javascript
function someHanlder() {}
$(document).on('click', PAGE_SPECIFIC_SELECTORS, someHanlder)
```

It's more expressive and you don't need worry about that the code is still executed in the page which are not related.  This can be removed as well I think `if (!$disbursementPage.length) { return; }`